tmp/service_at_location.csv

```diff
@@ -0,0 +1,521 @@
+,51556,27506,
+,,27511,
+,51561,27516,
+,,27521,
+,,27526,
+,51576,27531,
+,51586,27536,
+,,27546,
+,,27051,
+,93286,27056,
+,,27061,
+,,27066,
+,68081,27071,
+,,27081,
+,,27086,
+,50781,27091,
+,51136,27096,
+,50861,27101,
+,"32796,30066,30061,30056,30036,30021,29926,29866,29856,29801,29791",27106,
+,68096,27111,
+,50931,27116,
+,50981,27131,
+,,27136,
+,,27141,
+,,27151,
+,51036,27156,
+,"83176,83171",27161,
+,92921,27166,
+,,27171,
+,51226,27176,
+,,27181,
+,"99161,28856,28851,28846,28841,28836,28831,28776",27186,
+,,27191,
+,68091,27196,
+,,27201,
+,51241,27206,
+,,27211,
+,51281,27216,
+,,27221,
+,,27226,
+,,27231,
+,,27236,
+,,27241,
+,,27246,
+,,27251,
+,51301,27256,
+,51306,27261,
+,51316,27266,
+,,27271,
+,51321,27276,
+,,27281,
+,51346,27286,
+,51351,27291,
+,51356,27296,
+,36426,26801,
+,,26806,
+,36441,26811,
+,36346,26816,
+,36376,26821,
+,36571,26826,
+,,26831,
+,,26836,
+,,26841,
+,,26846,
+,36351,26851,
+,36356,26856,
+,,26861,
+,36361,26866,
+,,26871,
+,,26876,
+,37711,26881,
+,37736,26886,
+,,26891,
+,,26896,
+,,26901,
+,,26906,
+,,26911,
+,,26916,
+,,26921,
+,,26926,
+,,26931,
+,,26936,
+,,26946,
+,,26956,
+,76936,26961,
+,,26966,
+,,26971,
+,,26976,
+,,26981,
+,,26986,
+,,26991,
+,38291,26996,
+,77271,27001,
+,50206,27011,
+,50226,27016,
+,50241,27021,
+,,27026,
+,,27031,
+,,27036,
+,,27041,
+,50281,27046,
+,,26551,
+,,26556,
+,34951,26561,
+,,26566,
+,,26571,
+,,26576,
+,36006,26581,
+,36096,26586,
+,36186,26596,
+,111176,26601,
+,,26606,
+,,26611,
+,68426,26616,
+,,26621,
+,,26626,
+,,26631,
+,36421,26636,
+,36711,26641,
+,,26646,
+,,26651,
+,,26656,
+,36731,26661,
+,,26666,
+,36341,26671,
+,,26676,
+,,26681,
+,"32201,32191,32186",26686,
+,,26691,
+,37261,26696,
+,,26701,
+,"37651,37636",26706,
+,68101,26711,
+,,26716,
+,37291,26721,
+,37316,26726,
+,,26731,
+,37321,26736,
+,,26741,
+,,26746,
+,"101806,53301,53296",26751,
+,86981,26756,
+,"49961,49956,49951,49946",26761,
+,,26766,
+,,26771,
+,,26776,
+,37331,26781,
+,,26786,
+,93946,26791,
+,37336,26796,
+,,26301,
+,,26306,
+,,26311,
+,,26321,
+,,26326,
+,29416,26331,
+,34901,26336,
+,,26341,
+,29421,26346,
+,,26351,
+,,26356,
+,,26361,
+,,26366,
+,,26371,
+,,26376,
+,,26381,
+,,26386,
+,34916,26391,
+,35071,26396,
+,35081,26406,
+,,26411,
+,,26416,
+,"51911,36001",26421,
+,,26426,
+,36326,26431,
+,29201,26436,
+,36331,26441,
+,,26446,
+,"84361,84321",26451,
+,,26456,
+,37111,26461,
+,,26466,
+,,26471,
+,,26476,
+,,26481,
+,36336,26486,
+,,26491,
+,,26496,
+,,26501,
+,"35076,35066,29436",26506,
+,,26511,
+,"91161,37436",26516,
+,"38491,33851",26521,
+,,26526,
+,"49316,49311,49306,28706",26531,
+,,26536,
+,,26541,
+,,26546,
+,"30101,29631,29551,29546",26206,
+,"30146,30141,30136",26211,
+,"30211,30186,29601",26216,
+,"30226,29611",26221,
+,"30181,30176,29596,29591",26226,
+,"30156,30151,29581",26231,
+,"30131,30126,30121",26236,
+,"30171,30166,29586",26241,
+,"30256,30251,29556,29396",26246,
+,"30236,30231,28806",26251,
+,"51276,51261,51256,30096,30091,29541,29531",26261,
```

Comment:
We should add a way to split these up into multiple rows in the future, but this works for now :)